src/dotnet/Telemetry.cs

```diff
@@ -31,12 +42,26 @@ public Telemetry()
                 return;
             }
 
+            int sampleRate = ReciprocalSampleRateValue;
+            bool isCITestMachine = Env.GetEnvironmentVariableAsBool(ContinousIntegrationFlag);
```

Comment:
And do we EVER need telemetry from our CI builds?  They do the same thing over and over.